core/templates/pages/about-page/about-page.component.html

```diff
@@ -1,261 +1,1214 @@
+          <source type="image/webp" [srcset]="getStaticImageUrl('/about/books_dsk.webp')">
+          <source type="image/png" [srcset]="getStaticImageUrl('/about/books_dsk.png')">
+          <img [src]="getStaticImageUrl('/about/books_dsk.png')" alt="">
+        </picture>
+      </div>
+      <div class="oppia-about-section-six-right">
+        <h2 class="oppia-about-outcome-title" [innerHTML]="'I18N_ABOUT_PAGE_EASILY_CREATE_LESSON' | translate"></h2>
+        <p class="oppia-about-outcome-content" [innerHTML]="'I18N_ABOUT_PAGE_CREATE_LESSON_CONTENT' | translate"></p>
+        <div class="oppia-about-classroom-button-container">
+          <form action="<[$ctrl.classroomUrl]>">
+            <button type="submit" (click)="onClickCreateLessonButton()"
+                    class="btn oppia-about-button oppia-about-button-browse oppia-about-browse-button-position oppia-transition-200"
+                    [innerHTML]="'I18N_ACTION_CREATE_LESSON_BUTTON' | translate">
+            </button>
+          </form>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="oppia-about-page-section-seven">
+    <h2 class="get-started-title" [innerHTML]="'I18N_ABOUT_PAGE_SEC_SEVEN_TITLE' | translate"></h2>
+    <div class="oppia-about-classroom-button-container-one">
+      <form action="<[$ctrl.classroomUrl]>">
+        <button type="submit" (click)="onClickGuideForTeacherButton()"
+                class="btn oppia-about-button oppia-about-button-browse oppia-about-browse-button-position oppia-transition-200"
+                [innerHTML]="'I18N_ACTION_GUIDE_FOR_TEACHERS' | translate">
+        </button>
+      </form>
+    </div>
+    <div class="oppia-about-classroom-button-container-two">
+      <form action="<[$ctrl.classroomUrl]>">
+        <button type="submit" (click)="onClickTipsForParentsButton()"
+                class="btn oppia-about-button oppia-about-button-browse oppia-about-browse-button-position oppia-transition-200"
+                [innerHTML]="'I18N_ACTION_TIPS_FOR_PARENTS' | translate">
+        </button>
+      </form>
+    </div>
+    <div class="oppia-about-classroom-button-container-three">
+      <form action="<[$ctrl.classroomUrl]>">
+        <button type="submit" (click)="onClickVisitClassroomButton()"
+                class="btn oppia-about-button oppia-about-button-browse oppia-about-browse-button-position oppia-transition-200"
+                [innerHTML]="'I18N_ACTION_EXPLORE_LESSONS' | translate">
+        </button>
+      </form>
+    </div>
   </div>
 </div>
+
 <style>
-  .oppia-about-page-container .oppia-about-foundation-tab-anchor {
-    background-color: #00645c;
-    border-radius: 4px;
-    color: white;
-    letter-spacing: 0.7px;
-    width: 33%;
-  }
-  .oppia-about-page-container .oppia-about-foundation-tab-anchor:hover {
-    background-color: #3d9991;
-    text-decoration: none;
-  }
-  .oppia-about-page-container .oppia-about-text-align {
+  * {
+    margin: 0;
+  }
```

Comment:
Why is this needed?

Reply:
It had space between the sections, so to remove those spacings I had to add this.

Reply:
It seems this is a bit dangerous to do. Do you think you can introduce this on individual components instead?

Reply:
Done.